eng/Versions.props

```diff
@@ -48,9 +48,8 @@
     <MicrosoftExtensionsDependencyInjectionAutoActivationVersion>9.9.0</MicrosoftExtensionsDependencyInjectionAutoActivationVersion>
     <MicrosoftExtensionsDiagnosticsTestingVersion>9.9.0</MicrosoftExtensionsDiagnosticsTestingVersion>
     <MicrosoftExtensionsTimeProviderTestingVersion>9.9.0</MicrosoftExtensionsTimeProviderTestingVersion>
-    <!-- dotnet/runtime BCL compat packages for netfx -->
-    <MicrosoftBclMemoryVersion>9.0.9</MicrosoftBclMemoryVersion>
-    <MicrosoftBclTimeProviderVersion>9.0.9</MicrosoftBclTimeProviderVersion>
+    <MicrosoftExtensionsServiceDiscoveryVersion>9.5.0</MicrosoftExtensionsServiceDiscoveryVersion>
```

Comment:
@eerhardt we'll want to add these to dependency flow on Version.Details.xml as well as setup a subscription so we update these automatically in some cadence. How common is it for us to make changes to SD libraries and for Aspire to depend on those recently made changes? My guess is not common, but want to ask.

Reply:
We will want to update them on every release - just like the rest of the dotnet/extensions libraries.